content/docs/learning-guide/tutorials/assets/physx-colliders.md

```diff
@@ -68,9 +76,7 @@ The following table summarizes the most import information about the available c
 
 ## Generate PhysX collider assets
 
-You can generate PhysX collider assets from any source asset that contains at least one mesh. You can customize the settings for PhysX collider generation in Scene Settings, in the **PhysX** tab.
-
-If you are unfamiliar with Scene Settings, refer to the [Mesh Asset tutorial](../mesh-assets) and check the [Scene Settings PhysX Tab](/docs/user-guide/assets/scene-settings/physx-tab) topic for in-depth information on the options for generating PhysX collider assets.
+You can generate PhysX collider assets from any source asset that contains at least one mesh. You can customize the settings for PhysX collider generation in Scene Settings, in the **PhysX** tab. If you are unfamiliar with Scene Settings, refer to the [mesh processing tutorial](../mesh-assets) and check the [Scene Settings PhysX tab](/docs/user-guide/assets/scene-settings/physx-tab) topic for in-depth information on the options for generating PhysX collider assets.

```

Comment:
Is `../mesh-assets` a valid link in `[mesh processing tutorial](../mesh-assets)`? Normally I see a full path there.

Reply:
Yes, relative links are valid.